wp-content/themes/goonj-crm/admin-style.css

```diff
@@ -10,3 +10,6 @@
 #wpbody .wpbody-content #crm-container .crm-accordion-header:not(.crm-master-accordion-header)+.crm-accordion-body, .crm-container .crm-accordion-bold>.crm-accordion-body {
     box-shadow: none !important;
 }
+.crm-contact-actions-list-inner .crm-contact_activities-list .crm-activity-type_4 {
```

Comment:
I'm wondering what exactly does the number `4` represents in the class `crm-activity-type_4`. Is it the ID of the ActivityType Induction?

If it's activity type ID then we might have different classes in staging and prod

Reply:
No, this is not the activity type ID , In `crm-activity-type_4` here 4 represent the position and placement of the induction tab. I also checked in both staging and production, this is same there.